examples/sunmatrix/slunrloc/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ foreach(example_tuple ${sunmatrix_slunrloc_examples})
 
     target_include_directories(${example_target} PRIVATE . ..)
 
+    target_compile_features(${example_target} PRIVATE cxx_std_14)
```

Comment:
Shouldn't the default for `CMAKE_CXX_STANDARD` cover this?

Reply:
It should, but the GitLab CI keeps failing like it doesnt have C++14:  https://lc.llnl.gov/gitlab/sundials/sundials/-/jobs/1303952#L1562